.pre-commit-config.yaml

```
@@ -91,6 +91,32 @@ repos:
         args: [--enable=default-role]
         files: ^Doc/|^Misc/NEWS.d/
 
+  - repo: https://github.com/oxipng/oxipng
+    rev: v9.1.5
+    hooks:
+      - id: oxipng
+        args: [ "-o", "2", "--strip", "safe", "--alpha" ]
```

```suggestion
        args: [--opt=2, --strip=safe, --alpha]
```
Although `--opt=2` is the default so perhaps?
```suggestion
        args: [--strip=safe, --alpha]
```